legacy/src/main/java/io/github/notenoughupdates/moulconfig/gui/editors/GuiOptionEditorDropdown.java

```diff
@@ -129,17 +129,17 @@ public void renderOverlay(int x, int y, int width) {
             int dropdownHeight = 13 + 12 * values.length;
 
             int main = 0xff202026;
-            int blue = 0xff2355ad;
+            int outlineColour = 0xffffffff;
```

Comment:
This drop down colour is quite intense. I suppose it fits better with the rest of the ui but it reminds me to once again add support for custom color palettes.